vsc_pdf_template_transformer/lib/models/tpl_text.dart

```diff
@@ -66,4 +69,16 @@ class TplText implements wb.WidgetBuilder {
     print('--- value style: ${value.text.style!.color!.toHex()} ------');
     return value;
   }
+
+  String? _evaluateInput(String? text, Map<String, dynamic> data) {
+    if (text != null) {
+      final evaluator = const ExpressionEvaluator();
```

Comment:
We need to construct this as:
```dart
ExpressionEvaluator(memberAccessors: [
      // Allows "map.field" access.
      MemberAccessor.mapAccessor,
    ])
```

This allows us to write an expression like `data.phone_numbers[0].phone`. The MapAccessor  allows you to access Dart map keys as if they were properties of an object (a lot like javascript).